six/modules/python/six.sicd/tests/test_create_sicd_xml.py

```diff
@@ -677,36 +717,45 @@ def initRMA():
     rma.rmat.dopConeAngleRef = 5
     return rma
 
-def initRMCR():
+def initRMCR(rma):
     rmcr = makeScopedCopyableRMCR()
     for i in range(3):
 	rmcr.refPos[i] = i
 	rmcr.refVel[i] = i * 2
     rmcr.dopConeAngleRef = 9
+    rma.rmcr = rmcr
+    return rma
 
-def initINCA():
+def initINCA(rma):
     inca = makeScopedCopyableINCA()
     inca.timeCAPoly = Poly1D(3)
     for i in range(3):
 	inca.timeCAPoly[i] = i
     inca.rangeCA = 5
     inca.freqZero = 10
-    inca.dopplerRateScaleFactorPoly = Poly2D(3, 3)
+    inca.dopplerRateScaleFactorPoly = Poly2D(0, 3)
```

Comment:
Because version 0.4 of the schema expects it to look like a Poly1D. Although the doc says 2D, so now I’m thinking it might be a typo in the schema?

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Sunday, April 10, 2016 11:00 AM
To: ngageoint/six-library <six-library@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>
Subject: Re: [ngageoint/six-library] Regression testing (#57)


In six/modules/python/six.sicd/tests/test_create_sicd_xml.py<http://cp.mcafee.com/d/1jWVIi41ASyMVtCZSbTdTdIIczxNEVvvdLzCm6kjhOqejqdTHLe6zCVEVudEEIEFLCzBNxYS8RxIOuMVnZxg5aEqTjBPqG6JQVsSWVHCT7fZvATAnQjhOzRXBQSkT-vKqenbCzBPG8FHnjlKMVOEuvkzaT0QSyrjdTV5MQsL8EIFTudTdAVPmEBC4pj9JAenO17ifRB2ukA7BPV48X4JyPpV-fzwcgBiPtaiH0ly-qekmnS7DzsaNZoYI2ycFASO7bCS67QjobZ8Qg6BK17ifRB2ukA7BPV48X4Iqnjh02_8N-B9lwq89_isE4jh0qa0cv76yAS-CrBA9Y>:

>      inca = makeScopedCopyableINCA()

>      inca.timeCAPoly = Poly1D(3)

>      for i in range(3):

>       inca.timeCAPoly[i] = i

>      inca.rangeCA = 5

>      inca.freqZero = 10

> -    inca.dopplerRateScaleFactorPoly = Poly2D(3, 3)

> +    inca.dopplerRateScaleFactorPoly = Poly2D(0, 3)

Why was this changed..?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<http://cp.mcafee.com/d/k-Kr3zqb3BSrToLsTsSOMOe76zBZYS-epophd79EVdETuKYUqerCzBUSyyOyC-qen67Pozm6P9X3BvS50kGxHtendGEqTjBPrHCKrss_R-juhvhd7afnKnjpjvV-VEVsKqeneEyCJtdmX3DaxVZicHs3jqpJcTvAn3hOYyyODtUTsSjDdqymohBcCSgVv84t8_mk9VigunfAgzIiSbdDDU-eNSh_QOXdCnzhO-CxXb5HcLnujd7a5TL86zBwwyeKwVr_73xO06hvd7abbX3PNK5o-Ium1h6kOrp3BPr33W9I5-Aq83iT0zF7WOxfai3OVYy4tymdbFEw1vAo_iAGMd44_Fek29Ewd506fzzhirvjdXTIWhj1Z>


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.